atd-vzv/src/views/nav/SideMapTimeOfDayChart.js

```diff
@@ -33,36 +40,47 @@ export const SideMapTimeOfDayChart = ({ filters }) => {
       crashEndpointUrl,
       mapFilters,
       dateRange,
-      mapPolygon
+      mapPolygon,
+      fieldsToRequest
     );
+
     !!apiUrl &&
       axios.get(apiUrl).then((res) => {
-        setChartData(res.data);
+        setCrashes(res.data);
       });
   }, [dateRange, mapPolygon, mapFilters]);
 
   useMemo(() => {
-    const crashes = chartData;
     // When chartData is set, accumulate time window data
     if (!!crashes) {
-      const crashTimeWindowAccumulatorArray = Object.keys(filters).map(
-        (filter) => 0
+      // For each window of time in the config, add fatalities and serious injuries to initial count of 0
+      const crashTimeWindows = Object.values(timeWindowConfig).map(
+        (windowArray) => windowArray
+      );
+      const crashTimeWindowAccumulatorArray = Object.keys(timeWindowConfig).map(
+        () => 0
       );
-      const crashTimeWindows = Object.values(filters).map((filter) => filter);
+
       const crashTimeTotals = crashes.reduce((accumulator, crash) => {
         crashTimeWindows.forEach((timeWindow, i) => {
           const crashDate = crash.crash_date;
           const crashHour = parseInt(format(new Date(crashDate), "H"));
-          crashHour >= timeWindow[0] &&
-            crashHour <= timeWindow[1] &&
-            accumulator[i]++;
+
+          const crashFatalities = parseInt(crash.death_cnt);
+          const seriousInjuries = parseInt(crash.sus_serious_injry_cnt);
+          const isCrashInTimeWindow =
+            crashHour >= timeWindow[0] && crashHour <= timeWindow[1];
+
+          if (isCrashInTimeWindow) {
+            accumulator[i] = accumulator[i] + crashFatalities + seriousInjuries;
+          }
```

Comment:
We used to add up the total number of crash records per time of day window that returned using the map filters for severity, mode, and date range. 

Now, we add up the number of fatalities and serious injuries for each crash record.